types/react/index.d.ts

```diff
@@ -942,7 +942,7 @@ declare namespace React {
      * @see https://reactjs.org/docs/hooks-reference.html#useref
      */
     // TODO (TypeScript 3.0): <T extends unknown>
-    function useRef<T = undefined>(): MutableRefObject<T | undefined>;
+    function useRef<T = any>(): MutableRefObject<T | undefined | null>;
```

Comment:
The `= undefined` is there for a reason, it's to stop it from defaulting to `unknown`. Making it default to `any` is _much, much worse_.

Reply:
Your arguments are where exactly? I gave mine below

Reply:
Give me an example where it defaults to `unknown`. I gave some, imo, false positives below